utils/s3/client.cc

```diff
@@ -286,9 +288,36 @@ client::group_client& client::find_or_create_client() {
 }
 
 future<> client::make_request(http::request req, http::experimental::client::reply_handler handle, std::optional<http::reply::status_type> expected, seastar::abort_source* as) {
-    co_await authorize(req);
     auto& gc = find_or_create_client();
-    co_await gc.retryable_client.make_request(std::move(req), std::move(handle), expected, as);
+    bool renew_creds = false;
+    std::exception_ptr ex;
+    while (true) {
```

Comment:
This is horrible. The map_s3_client_error doesn't really _map_ an error on some expired-token special case just to let _this_ place uglify the retryably http client invocation. I've no ideas how to make it less horrible though

Reply:
Not that I'm happy with how I did it here, but wait, I think I have an idea to make it less ugly

Reply:
how about now?